cmd/config.go

```diff
@@ -90,6 +90,14 @@ func ConfigSet(appID string, configVars []string) error {
 		configMap["SSH_KEY"] = base64.StdEncoding.EncodeToString([]byte(sshKey))
 	}
 
+	// NOTE(bacongobbler): check if the user is using the old way to set healthchecks. If so,
+	// send them a deprecation notice.
+	if _, ok := configMap["HEALTHCHECK_URL"]; ok {
```

Comment:
I figured that since HEALTHCHECK_URL is the trigger to start the legacy healthcheck we only need to check for that one environment variable. Do you just want to check for any signs of `HEALTHCHECK_` in the `configMap`?